requirements.txt

```diff
@@ -0,0 +1 @@
+pyperclip
```

Comment:
Once we make pyperclip an optional dependency we can remove it from requirements.txt.